indico/modules/users/module.json

```diff
@@ -6,6 +6,7 @@
     "personal_data": "./PersonalDataForm.jsx",
     "favorites": "./Favorites.jsx",
     "data_export": "./DataExport.jsx",
-    "user_delete": "./UserDelete.jsx"
+    "user_delete": "./UserDelete.jsx",
+    "user_block": "./UserBlock.jsx"
```

Comment:
I suggest merging personal_data, user_delete and user_block into a single entry here; and since it's on the "personal data" page, I'd go for `"personal_data": "./personal_data.js",`

In that file, you can then simply import {PersonalDataForm,UserDelete,UserBlock}.jsx.